data/seeds/01-Users.js

```diff
@@ -9,7 +9,12 @@ exports.seed = function (knex) {
       address: 'Sweden',
       latitude: 60.8934744,
       longitude: -0.2821392,
-      photo: 'https://upload.wikimedia.org/wikipedia/en/e/e7/The_Swedish_Chef.jpg'
+      photo: 'https://upload.wikimedia.org/wikipedia/en/e/e7/The_Swedish_Chef.jpg',
+      pets: {},
+      allergens: {},
+      dietaryRestrictions: {},
+      dietaryPreferences: {},
+      children: {}
```

Comment:
why do we need to account for scalar values for these? I'm probably misunderstanding how it works.  I thought we were using array or array-like data to store these lists. Sorry if this is missing the mark, trying to understand the need for the changes.

Reply:
The new migration stores those values as a JSON object, so I figure we would be storing them similarly to the hashtags and modifiers where it would be like { pets: [ {pet 1}, {pet 2} ] }.

Reply:
thank you for helping me understand.  and adding these new modifiers so quickly